website/i18n/ja/docusaurus-plugin-content-docs/current/howdoesitwork.mdx

```diff
@@ -4,7 +4,7 @@ sidebar_position: 20
 
 # どうやって動いているの?
 
-Wailsは、webkitフロントエンドを備えた、何の変哲もないGoアプリです。 アプリ全体のうちGoの部分は、アプリのコードと、ウィンドウ制御などの便利な機能を提供するランタイムライブラリで構成されています。 フロントエンドはwebkitウィンドウであり、フロンドエンドアセットをウィンドウ上に表示します。 フロントエンドからも、Javascriptでランタイムライブラリを呼び出すことができます。 そして最終的に、Goのメソッドはフロントエンドにバインドされ、ローカルのJavascriptメソッドであるかのように、フロントエンドから呼び出すことができます。
```

Comment:
Are we reverting this from Japanese?

Reply:
I removed those auto-added machine translations and copied English translations when I added Japanese, I don't know how those were generated, but you were the translator and approver of those characters, so I removed them all and let Japanese translators re-translate to ensure translation accuracy.